src/extension.ts

```diff
@@ -646,4 +351,47 @@ function createIntegrationsView(): void {
 			await camelKIntegrationsProvider.refresh().catch(err => console.log(err));
 		}
 	});
-}
\ No newline at end of file
+}
+
+export async function installDependencies() {
+
+	let gotKamel : boolean = false;
+	await checkKamelCLIVersion().then ( (kamelCliVersion) => {
+		if (kamelCliVersion) {
+			shareMessageInMainOutputChannel(`Found Apache Camel K CLI (kamel) version ${kamelCliVersion}...`);
+			gotKamel = true;
+		}
+	}).catch ( () => { 
+		// ignore 
+	});
+	const kubectlCliVersion : string = await checkKubectlCLIVersion();
+	if (kubectlCliVersion) {
+		shareMessageInMainOutputChannel(`Found Kubernetes CLI (kubectl) version ${kubectlCliVersion}...`);
+	}
+
+	const minikubeCliVersion : string = await checkMinikubeCLIVersion();
```

Comment:
why checking minikube?
Minikube can deploy different versions of Kubernetes https://kubernetes.io/docs/setup/learning-environment/minikube/#specifying-the-kubernetes-version I would expect to have Camel K requiring a specific Kubernetes version range but not a specific minikube version.
Or there is really a specific minikube version?

Reply:
I've definitely hit the weird combination of minikube/kubectl/kamel cli tools - but we really don't need to check the minikube version for the tooling. It was more of a mental check than anything else I think.